infrastructure/ai/providers/anthropic_provider.py

```
@@ -137,12 +159,31 @@ async def generate(
 
             parts = []
             for block in response.content:
+                # 诊断日志：记录每个 block 的类型和内容
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)